src/components/test-selection/TestSelection.tsx

```diff
@@ -27,53 +22,30 @@ class TestSelection extends React.Component<Props> {
     },
   ];
 
-  public render() {
-    return (
-      <div className="test-selection">
-        <Navbar
-          className="test-selection__navbar"
-          backButton={true}
-          title={i18n("nav_test")}
-          onBackButtonClick={this.onNavbarBackButtonClick}
-        />
-
-        <div className="test-selection__entries">
-          {this.testEntries.map((testEntry, index) => (
-            <TestEntry
-              key={index}
-              {...testEntry}
-              onPracticeClick={this.onTestEntryPracticeListener(testEntry)}
-              onSettingsClick={this.onTestEntrySettingsListener(testEntry)}
-            />
-          ))}
-        </div>
+  const history = useHistory();
+
+  return (
+    <div className="test-selection">
+      <Navbar
+        className="test-selection__navbar"
+        backButton={true}
+        title={i18n("nav_test")}
+        // TODO: replace "push" with "replace" in the future
+        onBackButtonClick={() => history.push(MAIN_MENU)}
+      />
+
+      <div className="test-selection__entries">
+        {testEntries.map((testEntry, index) => (
+          <TestEntry
+            key={index}
```

Comment:
probably not a _real_ issue, but using `index` for `key` is generally frowned upon.  Perhaps using  `testEntry.testRoute` is more suitable as I assume the test routes should always be unique?

Reply:
In this case, it's not an issue because the list is static.

But yeah, `testEntry.testRoute` should be unique and better for this case.